model/chunks/chunkassignment.go

```diff
@@ -28,25 +47,29 @@ func (a *Assignment) Verifiers(chunk *flow.Chunk) flow.IdentifierList {
 // HasVerifier checks if a chunk is assigned to the given verifier
 // TODO: method should probably error if chunk has unknown index
 func (a *Assignment) HasVerifier(chunk *flow.Chunk, identifier flow.Identifier) bool {
```

Comment:
Could you address this TODO as part of the PR as well? There is only one caller.